frontend/src/Metamaps/PasteInput.js

```diff
@@ -70,11 +67,23 @@ const PasteInput = {
       Import.handleURL(text, coords)
     } else if (text[0] === '{') {
       Import.handleJSON(text)
-    } else if (text.match(/\t/)) {
+    } else if (text.match(/[Tt]opics\t/) || text.match(/[Ss]ynapses\t/)) {
       Import.handleTSV(text)
     } else {
-      // just try to see if CSV works
-      Import.handleCSV(text)
+      //Handle as plain text
+      let textItems = text.split("\n")
+      if (textItems.length === 1){
+        if(textItems[0].trim() !== ""){
+            Import.handleTEXT(textItems[0].trim(),coords)
+        }
+      }
+      else if (window.confirm('Are you sure you want to create ' + textItems.length + ' new topics?')) {
+        textItems.forEach(item => {
+          if(item.trim() !== ""){
+            self.handle(item.trim(),coords)
```

Comment:
Can you run eslint and/or check out https://codeclimate.com/github/metamaps/metamaps/pull/1109 to fix simple style stuff.